projects/schematics/src/shared/lib-configs/integration-libs/opf-schematics-config.ts

```diff
@@ -70,46 +160,92 @@ export const OPF_CHECKOUT_SCHEMATICS_CONFIG: SchematicConfig = {
     scssFileName: OPF_SCSS_FILE_NAME,
     importStyle: SPARTACUS_OPF,
   },
-  dependencyFeatures: [CHECKOUT_BASE_FEATURE_NAME, OPF_BASE_FEATURE_NAME],
-  importAfter: [
-    {
-      markerModuleName: CHECKOUT_BASE_MODULE,
-      featureModuleName: OPF_CHECKOUT_MODULE,
-    },
-  ],
   customConfig: buildOpfConfig,
+  dependencyFeatures: [
```

Comment:
It is the only way I found to force the installation of other 5 libs.
If you think about a better way, let me know.